README.md

```diff
@@ -1 +1,18 @@
-# Welcome to OpenBC Website
\ No newline at end of file
+# Welcome to OpenBC
+
+## Project Set Up Instructions
+
+1. Clone the repo `openbc-web` using SSH
+2. `cd` into the repo
+3. Type `node -v` to ensure you have the latest version of Node.js installed, install Node.js or update your version if needed
+4. From the top-level directory of the project directory `run npm install` to install the dev dependencies
+5. Type `npm run start` to start the application (development mode)
+6. To access the application navigate to `http://localhost:3000` in your web browser
+7. To exit the server `Command + C`
+
+## Git Flow
+
+1. `cd` into the repo
+2. `git fetch --all` to ensure your local git information is up to date
+3. Create a new feature branch `git checkout -b feature/newFeatureNameHere`
+4. Enter `git branch` to ensure you are on the correct feature branch
```

Comment:
not sure if it's important for the readme of this project. git flows are important to know for sure but that's a long topic( probably deserves its own document) and we should try to keep the project document short and sweet.

Reply:
I have been asking them to create branches with the following convention

features/<description of change>

I would not say we are strictly following git flow. I can see how the wording here might cause confusion.

Reply:
i see, i believe that information would go into contributing.md for guidelines on how contribute ? or the standard in terms of branch names etc? information about how to create branch is too much imo. just having the standard on branch name should suffice.

Reply:
@umsu2 , I agree

I think we need to put some design thought into this.

We plan to use README as the source of truth for project details. This will be part of our website.

I currently think we need the following docs
- README: Describe project and fill the information for our website
- Setup instructions
- Contribution instructions

Extra:
- Maybe README links to Setup and Contributions
- Maybe Setup and Contributions could be part of Github Wiki?

Reply:
@umsu2 ,

Not sure direction of this PR for now.

We could
1) Push this PR through and have those discussion at a later time
2) Leave this PR as is, have those discussions first

Reply:
i'd say let's get the mvp through first.